app/views/businesses/show.html.erb

```diff
@@ -1,72 +1,45 @@
-<div class="col">
+<div class="container" style="padding-bottom:100px; padding-top:10px;">
   <%= simple_form_for @business do |f| %>
-  <div class="show container">
-    <div class="row">
-      <h1 class="show-header"> Your Business Details</h1>
-    </div>
-
-    <div class="row">
-      <h4> <%= f.input :name, placeholder: @business.name %></h4>
-    </div> 
-
-    <hr/>
-
-    <div class="row">
-      <h4> <%= f.input :email, placeholder: @business.email %></h4>
-    </div>
-
-    <hr/>
-
-
-    <div class="row">
-      <h4> <%= f.input :phone_number, placeholder: @business.phone_number %></h4>
-    </div>
-
-    <hr>
-
-    <div class="row">
-      <h4> <%= f.input :business_url, placeholder: @business.business_url %></h4>
-    </div>
-  <div>
-  <%= f.button :submit, class: "btn btn-primary", data: { disable_with: "Saving..." } %>
+    <h3 style="text-align:center;"> Your Business Details</h3>
+    <div class="row" style="margin-bottom:20px; ">
+      <div class="col">
+        <%= f.input :name %>
+        <%= f.input :email %>
+      </div>
+      <div class="col">
+        <%= f.input :phone_number %>
+      </div>
+      <div class="col">
+        <%= f.input :business_url %><br>
+      </div>
+      <%= f.input :description, as: :trix_editor %>
+    </div>
+    <%= f.button :submit, class: "btn btn-sm btn-primary", data: { disable_with: "Saving..." } %>
   <% end %>
-    <div class="col">
-    <h1>Preview Customization</h1>
-  <iframe src="http://localhost:3000/<%= @business.business_url %>" style="width: 100%; height: 500px;"
-      <p>Placeholder text; only shows up if the page DOESN'T render!</p>>
```

Comment:
If it's possible the page won't render, you might want a better message that this.  Maybe... Oops, something went wrong.